app/src/views/features/sessions/SessionViewer/SessionDetails.tsx

```diff
@@ -129,6 +129,13 @@ const SessionDetails: React.FC<SessionDetailsProps> = ({ isInsideIframe = false
       return;
     }
 
+    // no rrweb listener on the player works when focus is shifted from the tab
+    // The player keeps playing even when the tab is not in focus.
+    // so we add a listener on the window to pause the player when the tab is blurred
+    window.addEventListener("blur", () => {
```

Comment:
Is it possible that on component remount, we might attach multiple such listeners on `window`, as we are not calling `removeEventListener` here?

Reply:
Yes. It will happen. Removing listener on unmount.